tests/copp/test_copp.py

```diff
@@ -533,6 +533,14 @@ def _setup_testbed(dut, creds, ptf, test_params, tbinfo, upStreamDuthost, is_bac
     logging.info("Update the rate limit for the COPP policer")
     copp_utils.limit_policer(dut, rate_limit, test_params.nn_target_namespace, test_params.neighbor_miss_trap_supported)
 
+    if not is_backend_topology:
+        # make sure traffic goes over management port by shutdown bgp toward upstream neigh that gives default route
+        upStreamDuthost.command("sudo config bgp shutdown all")
```

Comment:
There was a `sleep(30)` in previous code. Is it safe to remove it?
I'm considering if we should add a `wait_until` here to confirm all BGP sessions are down before preceed.